viv-wallet-app/src/services/login.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line camelcase
```

Comment:
can we rename when we import? it avoids the `disable`...

Reply:
I don't even know why I did not do that in the first place. 🤣